platforms/core-runtime/serialization/build.gradle.kts

```diff
@@ -27,7 +27,6 @@ dependencies {
     api(projects.hashing)
     api(projects.javaLanguageExtensions)
 
-    api(libs.fastutil)
```

Comment:
💭 This is the goal of the PR, so Maven cache extension does not have a conflict on `fastutil`.